eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxos.hs

```diff
@@ -178,7 +179,7 @@ scriptsValidateTransition = do
 
   () <- pure $! traceEvent validBegin ()
 
-  case collectTwoPhaseScriptInputs ei sysSt pp tx utxo of
+  case collectTwoPhaseScriptInputs (unsafeLinearExtendEpochInfo slot ei) sysSt pp tx utxo of
```

Comment:
I think we need to switch on `allowOutsideForecastTTL pp` here as well, so that `unsafeLinearExtendEpochInfo` is only used before major version 7